lesson11/index.js

```diff
@@ -0,0 +1,55 @@
+const ul = document.getElementById('js-ul');
+
+function addLoading() {
+  const loading = document.createElement('img');
+  loading.src = "./loading-circle.gif";
+  loading.id = "loading";
+  ul.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+async function getData() {
+  addLoading();
+  try {
+    const response = await fetch("https://jsondata.okiba.me/v1/json/eGNos210920102817");
+    if (!response.ok) {
+      throw new Error(response.statusText);
+    } else {
+      return response.json();
+    }
+  } catch (e) {
+    const div = document.getElementById('js-wrapper');
+    div.textContent = "エラーが発生しました！";
+    console.error(e);
+  } 
+}
+
+
+async function createLists() {
+  const responseData = await getData();
```

Comment:
if response is undefined,what do you do.

Reply:
Thank you for your review.

Added a process to display text in the browser if responseData is undefined. 
Please check it.